pkg/symbol/symbol.go

```diff
@@ -80,6 +80,9 @@ func (s *Symbolizer) Symbolize(prof *profiler.Profile) error {
 	if s.vdsoCache != nil {
 		for _, l := range prof.UserLocations {
 			// TODO(kakkoyun): Or Use Mapping.Pathname
+			if l.Location.Mapping == nil {
```

Comment:
Occasionally some mappings are `nil`, I am not sure why this can happen just yet

Reply:
I can see two scenarios:

1) something about the stack walking went wrong and we have an address that is outside any real mappings
2) JITed section

Reply:
That could make sense! I am wondering about the actual condition that's causing this to dereference a `nil` pointer in the integration tests but in production, we haven't seen any crashes with this stack 🤔

Reply:
Could we add a metric about this? Like an `address out-of-range`.counter. Or do we already have one?
It would be better if we could distinguish if it's a JITed address.